articles/uploading-images-to-cloudinary-from-django-application/index.md

```diff
@@ -0,0 +1,270 @@
+Sometimes while working on applications that require the use of images, a simple process like uploading images to a server can become difficult.
+If you deploy your Django to platform like heroku, you can't save image.
+
+In this tutorial, we look at how to save images in Django using CLoudinary.
+
+Cloudinary is an end-to-end image- and video-management solution for websites and mobile apps, covering everything from image and video uploads, storage, manipulations, optimizations to delivery.
```

Comment:
In addition to this, please also add what other solutions are present, how it is better than others, what is the current method used, and if possible give advantages of using this.

In overall, you will have to give more details in the introductory paragraph and the conclusion.